x/did/spec/05_events.md

```diff
@@ -0,0 +1,15 @@
+# Events
+
+The did module emits the following events:
+
+## EndBlocker
```

Comment:
All events were placed under "EndBlocker" title, but none of the events are generated from the EndBlocker. The events are supposed to be categorised based on the message or function from where they get generated.

Reply:
removed the endblocker, placed under handler

Reply:
EndBlocker title can be removed